pkg/cmd/controller/unidling.go

```diff
@@ -39,6 +40,7 @@ func RunUnidlingController(ctx *ControllerContext) (bool, error) {
 		coreClient,
 		resyncPeriod,
 	)
+	unidlingMetrics.GetEventsTotalCounter()
```

Comment:
Since `NewUnidlingController` calls `GetEventsTotalCounter`, is this necessary?

Reply:
No - removed.